tests/scadnano_tests.py

```diff
@@ -671,19 +671,26 @@ def test_circular_auto_staple_hex(self) -> None:
         # To help with debugging, uncomment these lines to write out the
         # scadnano file
         #
-        # design.write_scadnano_file(directory=self.output_path,
-        #                            filename=f'{file_name}.{sc.default_scadnano_file_extension}')
+        design.write_scadnano_file(directory=self.output_path,
+                                   filename=f'{file_name}.{sc.default_scadnano_file_extension}')
 
     def test_paranemic_crossover(self) -> None:
-        file_name = "test_paranemic_crossover"
+        file_name = "test_crossover_to_same_helix"

```

Comment:
Is this a typo? Seems like file name should be "test_paranemic_crossover" instead of "test_crossover_to_same_helix"